README.md

```diff
@@ -464,7 +464,7 @@ application.
 
 ### On FreeBSD:
 
-The project can be built from scratch by following instructions for Linux above(but use `gmake` instead of `make`). 
```

Comment:
There's a rule here, that such changes (removing whitespaces in a line, where you don't change the merit)  aren't welcome in a PR.